src/Sulu/Bundle/PreviewBundle/Preview/Renderer/PreviewKernel.php

```diff
@@ -34,9 +35,13 @@ public function registerContainerConfiguration(LoaderInterface $loader)
         parent::registerContainerConfiguration($loader);
 
         if (in_array($this->getEnvironment(), ['dev', 'test'])) {
-            $loader->load(
-                implode(DIRECTORY_SEPARATOR, [__DIR__, '..', '..', 'Resources', 'config', 'config_preview_dev.yml'])
-            );
+            $loader->load(function(ContainerBuilder $container) use ($loader) {
+                if ($container->hasExtension('web_profiler')) {
```

Comment:
I also don't really understand what this is doing... Why are we nesting the `load` calls? :thinking: And if we check for the existence of the `web_profiler`, couldn't we remove the environment check above? Then this would even work if somebody wants to activate the `web_profiler` in production for some reason.

Reply:
the nested load is needed as its the only way to access the `$container` here correctly.
And yes you are right I did remove the environment check.